lib/elixir/lib/supervisor/spec.ex

```diff
@@ -75,6 +75,10 @@ defmodule Supervisor.Spec do
     * `:transient` - the child process is restarted only if it
       terminates abnormally, i.e., with an exit reason other than
       `:normal`, `:shutdown` or `{:shutdown, term}`
+  
+  Notice that supervisor that reached `max_restart_intensity` will exit with `:shutdown` reason. 
```

Comment:
Why is `max_restart_intensity` in backticks? There is no such argument, variable or option. Maybe it should be written literally as max restart intensity?